Add Github desktop alternative to readme (#2)
utd-sgdp/SGDP-2023: README.md

### Prerequisites

- [Unity Hub](https://unity.com/) with [Unity 2021.3.3f1](https://unity3d.com/get-unity/download/archive) installed

> - [GitKraken Client](https://www.gitkraken.com/), [Github Desktop](https://desktop.github.com/), or another Git client

Reviewer: ```suggestion
[GitKraken Client](https://www.gitkraken.com/), [GitHub Desktop](https://desktop.github.com/), or another Git client
```
Should be "GitHub Desktop".

### Contributing

> To start, create a branch in GitKraken or Github Desktop by clicking the "Branch" button. Then, enter a name for your branch and click "Create Branch". You can now make changes to the project. Once you are done, you can commit your changes by clicking the "Commit" button. Then, enter a message for your commit and click "Commit to Branch". You can now push your changes to the repository by clicking the "Push" button.

Reviewer: ```suggestion
To start, create a branch in GitKraken by clicking the "Branch" button. Then, enter a name for your branch and click "Create Branch". You can now make changes to the project. Once you are done, you can commit your changes by clicking the "Commit" button. Then, enter a message for your commit and click "Commit to Branch". You can now push your changes to the repository by clicking the "Push" button.
```
GitHub Desktop uses a different setup for creating and managing branches, so these steps are invalid.

Author: It uses the same thing, I checked. Or it's similar enough to where it's not confusing

Reviewer: ```suggestion
To start, create a branch in GitKraken by clicking the "Branch" button. Then, enter a name for your branch and click "Create Branch". You can now make changes to the project. Once you are done, you can commit your changes by clicking the "Commit" button. Then, enter a message for your commit and click "Commit to Branch". You can now push your changes to the repository by clicking the "Push" button. Similar steps can be followed using GitHub Desktop.
```
I'd rather make sure our documentation is as specific as possible. Freshmen and people unfamiliar with version control may be working on this project. I'm okay with just saying that similar steps exist for GitHub Desktop if that works with you.
